cpg_workflows/metamist.py

```diff
@@ -127,10 +127,10 @@
 )
 
 
-_metamist: Optional['Metamist'] = None
+_metamist: Optional["Metamist"] = None
```

Comment:
this file has been reformatted to have double instead of single quotes - I don't think there's any changes, but hard to tell at a glance. can you reformat it with `ruff format --config "format.quote-style = 'single'"` to be consistent with the rest of the repo?

Reply:
Sure